app/components/sidebar/sidebar_component.yml

```
@@ -0,0 +1,2 @@
+pl:
+  
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)